CleverRaven/Cataclysm-DDA
# Add scent eliminator spray

#### Summary
Content "Add scent eliminator spray"


#### Purpose of change


#53251

#### Describe the solution



Adds scent eliminator based on [this product](https://www.cabelas.com/shop/en/wildlife-research-center-super-charged-scent-killer-scent-elimination-spray-combo).
- Currently reduces scent completely for normal humans (-500. For context Strong smell trait is 800. Open to suggestions)
- Does not work on non-human scent
- Spawns with other hunting items
- Each application lasts a day

#### Describe alternatives you've considered


- Considered a new scent type, but it seems like these products work based on odor elimination rather than scent masking
- Should `effect_masked_scent` have a decay?

#### Testing


- Spawn scent eliminator
- Apply to self
- Confirm with scent map that scent goes away

#### Additional context
